databases/week3/mealsharing_database.sql

```diff
@@ -0,0 +1,36 @@
+DROP  DATABASE mealSharing;
+CREATE  DATABASE mealSharing;
+USE  mealSharing;
+
+CREATE TABLE `Meal` (
+  `id` int  unsigned auto_increment  primary key ,
+  `title` varchar(255) NOT NULL ,
+  `description` text NOT NULL ,
+  `location` varchar(255) NOT NULL ,
+  `time` datetime NOT NULL ,
+  `max_reservations` int NOT NULL ,
+  `price` decimal ,          
+  `created_date` datetime NOT NULL default CURRENT_TIMESTAMP()
+);
+
+CREATE TABLE `Reservation` (
+  `id` int unsigned auto_increment  primary key ,
+  `number_of_guests` int NOT NULL  ,
+  `meal_id` int unsigned NOT NULL   ,
+  `created_date` DATETIME NOT NULL  default CURRENT_TIMESTAMP() ,
```

Comment:
For default you can also use NOW() instead of CURRENT_TIMESTAMP(), they give you the same result :)